prototype_source/pt2e_quant_qat_static.rst

```diff
@@ -0,0 +1,470 @@
+            sampler=train_sampler)
+
+        data_loader_test = torch.utils.data.DataLoader(
+            dataset_test, batch_size=eval_batch_size,
+            sampler=test_sampler)
+
+        return data_loader, data_loader_test
+
+    def train_one_epoch(model, criterion, optimizer, data_loader, device, ntrain_batches):
+        # Note: do not call model.train() here, since this doesn't work on an exported model.
+        # Instead, call `torch.ao.quantization.move_exported_model_to_train(model)`, which will
+        # be added in the near future
+        top1 = AverageMeter('Acc@1', ':6.2f')
+        top5 = AverageMeter('Acc@5', ':6.2f')
+        avgloss = AverageMeter('Loss', '1.5f')
+    
+        cnt = 0
+        for image, target in data_loader:
+            start_time = time.time()
+            print('.', end = '')
+            cnt += 1
+            image, target = image.to(device), target.to(device)
+            output = model(image)
+            loss = criterion(output, target)
+            optimizer.zero_grad()
+            loss.backward()
+            optimizer.step()
+            acc1, acc5 = accuracy(output, target, topk=(1, 5))
+            top1.update(acc1[0], image.size(0))
+            top5.update(acc5[0], image.size(0))
+            avgloss.update(loss, image.size(0))
+            if cnt >= ntrain_batches:
+                print('Loss', avgloss.avg)
+    
+                print('Training: * Acc@1 {top1.avg:.3f} Acc@5 {top5.avg:.3f}'
+                      .format(top1=top1, top5=top5))
+                return
+    
+        print('Full imagenet train set:  * Acc@1 {top1.global_avg:.3f} Acc@5 {top5.global_avg:.3f}'
+              .format(top1=top1, top5=top5))
+        return
+
+    data_path = '~/.data/imagenet'
+    saved_model_dir = 'data/'
+    float_model_file = 'resnet18_pretrained_float.pth'
+
+    train_batch_size = 32
+    eval_batch_size = 32
+
+    data_loader, data_loader_test = prepare_data_loaders(data_path)
+    example_inputs = (next(iter(data_loader))[0])
+    criterion = nn.CrossEntropyLoss()
+    float_model = load_model(saved_model_dir + float_model_file).to("cuda")
+
+
+Export the model with torch.export
+----------------------------------
+
+Here is how you can use ``torch.export`` to export the model:
+
+.. code:: python
+
+    from torch._export import capture_pre_autograd_graph
+
+    example_inputs = (torch.rand(2, 3, 224, 224),)
+    exported_model = capture_pre_autograd_graph(float_model, example_inputs)
+
+
+.. code:: python
+
+    # or, to capture with dynamic dimensions:
+    from torch._export import dynamic_dim
+
+    example_inputs = (torch.rand(2, 3, 224, 224),)
+    exported_model = capture_pre_autograd_graph(
+        float_model,
+        example_inputs,
+        constraints=[dynamic_dim(example_inputs[0], 0)],
+    )
+
+``capture_pre_autograd_graph`` is a short term API, it will be updated to use the offical ``torch.export`` API when that is ready.
```

Comment:
```suggestion
.. note::

   ``capture_pre_autograd_graph`` is a short term API, it will be updated to use the offical ``torch.export`` API when that is ready.
```